# HG changeset patch
# User Rik <rik@octave.org>
# Date 1374276725 25200
#      Fri Jul 19 16:32:05 2013 -0700
# Node ID 960ac70ac8f37b4cb9836b05637553b49375c03e
# Parent  35a565fed4bd7b9ade0af53d5e316f2e9cc01320
prefdir.m: Adjust seealso references in docstring.

diff --git a/scripts/prefs/prefdir.m b/scripts/prefs/prefdir.m
--- a/scripts/prefs/prefdir.m
+++ b/scripts/prefs/prefdir.m
@@ -27,17 +27,17 @@
 ## @example
 ## prefdir
 ## @end example
 ##
 ## Change to the preferences folder
 ## @example
 ## cd (prefdir)
 ## @end example
-## @seealso{preferences, getpref, setpref, addpref, rmpref}
+## @seealso{getpref, setpref, addpref, rmpref, ispref}
 ## @end deftypefn
 
 ## Author: John Donoghue
 ## Version: 0.01
 
 function folder = prefdir ()
 
   folder = getenv ("HOME");
